docs/install-kubernetes.md

```diff
@@ -137,10 +143,14 @@ Your applications are now available in your local browser:
 - Jupyter-notebook - http://localhost:30040
 - Nuclio - http://localhost:30050
 - MLRun UI - http://localhost:30060
-- MLRun API (external) - http://localhost:30070
+- MLRun API (external) - http://localhost:30070 (or health check via http://localhost:30070/api/healthz)
 
 
 ```{admonition} Note
+You can check current state of installation via command `kubectl -n mlrun get pods`, where the main information
+is in columns `Ready` and `State`. If all images have already been pulled locally, typically it will take 
+a minute for all services to start.
+
```

Comment:
I think it is preferable to separate the notes into two,  the existing one that describes how to configure the k8s if it doesn't exposed on local host and the new one that explains how to determine whether all services are operational.